src/ai/backend/client/func/vfolder.py

```diff
@@ -166,6 +167,7 @@ async def download(
         basedir: Union[str, Path] = None,
         chunk_size: int = DEFAULT_CHUNK_SIZE,
         show_progress: bool = False,
+        address_map: Mapping[str, str] = get_config().address_map,
```

Comment:
Here is the same analogy with the Click command functions. It is better to use a static, immutable constant as the default value of keyword arguments.
For a functional interface like here, we could just use `None` as the default and assign `get_config().address_map` **inside** the function if the argument is `None`.